### PR TITLE
Destination geolocation

### PR DESCRIPTION
**Pull request by mrthegreensky**
With this pull request you can:
- [x] Add geolocation to destinations
- [x] Edit geolocations on destinations
- [x] Users have a set home location to Edmonton Alberta, because it's the best place ever.

### CODE REVIEW

File: Team9Project/src/com/indragie/cmput301as1/DestinationAddActivity.java [hunk 1]
```diff
@@ -1,33 +1,143 @@
...
+	public static final String ADD_TO_CLAIM = "com.indragie.cmput301as1.ADD_TO_CLAIM";
+	
+	//================================================================================
+	// Properties
+	//================================================================================
+	
+	/**
+	 * Field for name of the destination.
+	 */
+	protected EditText nameField;
```
Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/DestinationAddActivity.java:59]: Is there a reason for these not to be `private` instead? Is there a subclass of this class?

File: Team9Project/src/com/indragie/cmput301as1/DestinationAddActivity.java [hunk 2]
```diff
@@ -1,33 +1,143 @@
...
 	@Override
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
+		
+		setUpFields();
+		setUpListener();
+		
+		Boolean addToClaim = getIntent().getBooleanExtra(ADD_TO_CLAIM, false);
+		
+		if(addToClaim) {
```
Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/DestinationAddActivity.java]: Space after if

File: Team9Project/src/com/indragie/cmput301as1/DestinationEditActivity.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+package com.indragie.cmput301as1;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+/**
+ * Activity that presents a user interface for entering information to 
+ * edit a destination on a claim.
+ */
+public class DestinationEditActivity extends DestinationAddActivity {
```
Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/DestinationEditActivity.java:27]: Oh, so there is a subclass. Ignore my previous comment.

File: Team9Project/src/com/indragie/cmput301as1/DestinationAddActivity.java [hunk 4]
```diff
@@ -1,33 +1,143 @@
...
-	Button addLocationButton;
-	//TODO: get something back from the button
 	@Override
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
+		
+		setUpFields();
+		setUpListener();
+		
+		Boolean addToClaim = getIntent().getBooleanExtra(ADD_TO_CLAIM, false);
```
Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/DestinationAddActivity.java]: Use the primitive `boolean` type for consistency here

File: Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java [hunk 5]
```diff
@@ -64,6 +64,16 @@
 	public static final String EXTRA_FILTERED_EXPENSE_CLAIM_INDEX = "com.indragie.cmput301as1.EXTRA_FILTERED_EXPENSE_CLAIM_INDEX";
 	
 	/**
+	 * Request code for starting the {@link DestinationAddActivity}
+	 */
+	private static final int ADD_DESTINATION_REQUEST = 7;
```
Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java]: Number these starting at 30 like the other ones

File: Team9Project/src/com/indragie/cmput301as1/UserSettingsActivity.java [hunk 6]
```diff
@@ -86,6 +86,9 @@ protected void onCreate(Bundle savedInstanceState) {
 			}
 		} else {
 			setTitle(R.string.title_activity_user_settings_first_run);
+			homeLocation = new Geolocation(53.333333333, 113.5000000);
```
Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/UserSettingsActivity.java:92]: Comment here explaining what this location is

**Comment by indragiek** [comment]
Other than those minor things it looks good!

File: Team9Project/src/com/indragie/cmput301as1/DestinationAddActivity.java (hunk 1 above)
Comment (mrthegreensky) [inline Team9Project/src/com/indragie/cmput301as1/DestinationAddActivity.java:59]: Yup, DestinationEditActivity extends off this class.

**Pull request merged**